src/api/app/models/concerns/workflow_run_gitlab_payload.rb

```diff
@@ -43,6 +43,10 @@ def gitlab_target_branch
     payload[:after] # for a tag push event
   end
 
+  def gitlab_object_kind
```

Comment:
This makes little sense as it's no universal and it's just *always* `merge_request`...

Reply:
true, its dropped now